Source/RealismOverhaul.csproj

```diff
@@ -38,6 +38,11 @@
     <Compile Include="ModuleROSolarPanel.cs" />
     <Compile Include="ModuleEVADrag.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="DifficultySettings.cs" />
+    <Compile Include="DynamicPartHider\DynamicPartHider.cs" />
```

Comment:
Why 2 different directories if these things are supposed to work together?

Reply:
If another filter (US vs USSR) gets added, the new system might not want to be in the same directory

Reply:
Decided against this, will merge into the same directory